### PR TITLE
feat: Add Python analogues of Java templates

### PR DESCRIPTION
**Pull request by ch3zych3z**

### CODE REVIEW

**Review by DaniilStepanov — CHANGES_REQUESTED** [review]
Globally do not use ~[BODY]~ in python templates
And add empty_line to each file if not added

File: languages/python/sensitivity/lambdas/helpers/UnaryOpMutationInterface.tmt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+~class UnaryOpMutationInterface start~
```
[the remark this answers is not in the document]
Comment (ch3zych3z) [reply languages/python/sensitivity/lambdas/helpers/UnaryOpMutationInterface.tmt:1]: Why `fun` is assigned to `self._mutate_func`? With given realization of `mutate` `fun` won't be called

File: languages/python/sensitivity/virtuality/helpers/DerivedBinaryOpClass.tmt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+~class DerivedBinaryOpClass1 start~
```
[the remark this answers is not in the document]
Comment (ch3zych3z) [reply languages/python/sensitivity/virtuality/helpers/DerivedBinaryOpClass.tmt:1]: What are the rules of injected classes import? It was discussed at the meetings that there's no knowledge of locations where injected classes will be placed

File: languages/python/sensitivity/virtuality/helpers/ImplBinaryOpInterfaceClass.tmt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+~class ImplBinaryOpInterfaceClass1 start~
```
[the remark this answers is not in the document]
Comment (ch3zych3z) [reply languages/python/sensitivity/virtuality/helpers/ImplBinaryOpInterfaceClass.tmt:1]: Same question as at `DerivedBinaryOpClass1.tmt`

**Comment by ch3zych3z** [comment]
> Globally do not use ~[BODY]~ in python templates And add empty_line to each file if not added

What does it mean? Should templates with \~[BODY]\~ be removed?

Also replied in review discussion. Unreplied ones are fixed

File: languages/python/sensitivity/lambdas/helpers/UnaryOpMutationInterface.tmt (hunk 1 above)
[the remark this answers is not in the document]
Comment (DaniilStepanov) [reply languages/python/sensitivity/lambdas/helpers/UnaryOpMutationInterface.tmt:1]: ok. no fix needed

File: languages/python/sensitivity/virtuality/helpers/DerivedBinaryOpClass.tmt (hunk 2 above)
[the remark this answers is not in the document]
Comment (DaniilStepanov) [reply languages/python/sensitivity/virtuality/helpers/DerivedBinaryOpClass.tmt:1]: They will be located in the same directory with chosen file for mutation

File: languages/python/sensitivity/virtuality/helpers/ImplBinaryOpInterfaceClass.tmt (hunk 3 above)
[the remark this answers is not in the document]
Comment (DaniilStepanov) [reply languages/python/sensitivity/virtuality/helpers/ImplBinaryOpInterfaceClass.tmt:1]: Same answer

**Comment by DaniilStepanov** [comment]
> > Globally do not use ~[BODY]~ in python templates And add empty_line to each file if not added
> 
> What does it mean? Should templates with ~[BODY]~ be removed? Are imports from `typing` are needed? Why not just `from typing import *` for almost every injected file?
> 
> Also replied in review discussions. Unreplied ones are fixed

1. No, just delete ~[BODY]~ constructions
2. Imports from typing are needed, because we are not modifying helper classes

**Review by DaniilStepanov — APPROVED** [review]

**Pull request merged**